src/main/resources/application.yaml

```diff
@@ -377,6 +377,11 @@ notification:
         letterId: 5745f77c-6512-4082-8c34-63851f24eab1
       appointee:
         letterId: 114cadaa-7760-4699-9add-d3a252f68909
+    reviewConfidentialityRequest:
```

Comment:
should there be an appointee be here?

Reply:
No confidentiality always goes to appellant or joint party